examples/java-matter-controller/java/src/com/matter/controller/commands/discover/DiscoverCommissionablesCommand.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package com.matter.controller.commands.discover
+
+import chip.devicecontroller.ChipDeviceController
+import chip.devicecontroller.DiscoveredDevice
+import com.matter.controller.commands.common.CredentialsIssuer
+import com.matter.controller.commands.common.MatterCommand
+import java.util.concurrent.TimeUnit
+
+class DiscoverCommissionablesCommand(
+  controller: ChipDeviceController, credsIssuer: CredentialsIssuer?
+) : MatterCommand(controller, "commissionables", credsIssuer) {
+  private val MAX_DISCOVERED_DEVICES = 10
+  private val TIME_TO_WAIT_FOR_RESULTS_SECONDS: Long = 7
```

Comment:
These constants should go outside the class instance. Either at the top-level, or in a companion object. 

```
private const val MAX_DISCOVERED_DEVICES = 10
private const val TIME_TO_WAIT_FOR_RESULTS_SECONDS = 7L

class DiscoverCommissionablesCommand(...)
```

or:

```
class DiscoverCommissionablesCommand(...) {
  // other instance code

  companion object {
    private const val MAX_DISCOVERED_DEVICES = 10
    private const val TIME_TO_WAIT_FOR_RESULTS_SECONDS = 7L
  }
}
```